frontend/src/metabase/setup/components/LicenseTokenStep/LicenseTokenForm.module.css

```diff
@@ -0,0 +1,34 @@
+.breakoutHorizontalRule {
+  width: 100%;
+  position: absolute;
+  left: 0;
+  height: 1px;
+  background-color: var(--mb-color-border);
+}
+
+.breakoutHorizontalRuleContainer {
+  padding-top: var(--mantine-spacing-xl);
+  padding-bottom: var(--mantine-spacing-md);
+}
+
+.popoverContent {
+  display: flex;
+  width: 300px;
+  padding: var(--mantine-spacing-md);
+  flex-direction: column;
+  align-items: flex-start;
+  gap: var(--mantine-spacing-xs);
+
+  /* our --default-border-radius is 8px but the design system uses a custom value */
+  border-radius: 6px;
+  /* our --mb-color-border is eeecec but the design system uses a different value */
+  border: 1px solid #f0f0f0;
+  background: var(--mb-base-color-white);
```

Comment:
we should never use base colors except when defining semantic colors.

https://github.com/metabase/metabase/blob/8150ee9bbdc6e570d074f32edeef69c0db299e1b/frontend/src/metabase/css/core/colors.module.css#L14-L19

Reply:
Right. Removed. Reply: https://github.com/metabase/metabase/pull/59464#discussion_r2161084257